pkg/openapi2mcp/openapi2mcp.go

```diff
@@ -0,0 +1,383 @@
+package openapi2mcp
```

Comment:
How does it compare to this one? https://github.com/jedisct1/openapi-mcp/tree/main/pkg/openapi2mcp

Reply:
above ones seems to have too much boilerplate.